mod/groups/views/default/groups/invitationrequests.php

```diff
@@ -46,6 +63,14 @@
 		}
 	}
 	echo '</ul>';
+
+	if (!empty($count)) {
+		echo elgg_view('navigation/pagination', array(
```

Comment:
I mean it's not pressing, but I wonder how this never came up as an issue on large sites.